docs: Flex manual, update E-stop and module behavior table (#18857)
OpenTrons/opentrons-api: docs/flex-manual/docs/system-description.md

                <ul>
                    <li>Vertical motion is halted.</li>
                    <li>Motor brake on vertical axis is engaged to prevent the gripper from falling.</li>
                    <li>Jaw motors that exert gripping force remain enabled, so the gripper will not drop labware it may be carrying.</li>
                </ul>
            </td>
        </tr>
        <tr>
            <td><strong>Heater-Shaker Module</strong></td>
            <td>
                <ul>
                    <li>The shaker stops and homes.</li>
                    <li>The labware latch opens.</li>
                    <li>Heating is disabled.</li>
                </ul>
            </td>
        </tr>
        <tr>
            <td><strong>HEPA/UV Module</strong></td>
            <td>
                <ul>
                    <li>The UV lights and fan continue to operate.</li>
                    <li>Opening the door disables the lights.</li>
                </ul>
            </td>
        </tr>
        <tr>
            <td><strong>Plate Reader</strong></td>
            <td>
                <ul>
                    <li>When analyzing a sample, the module will continue to operate.</li>

>                     <li>If the gripper is moving the plate reader's lid, that module behaves as described above.</li>

Reviewer: Holding the line against calling the gripper a module.
```suggestion
                    <li>If the gripper is moving the plate reader's lid, its jaws remain enabled to hold the lid.</li>
```

>                     <li>The gripper will also place the plate reader's lid back in its docking area after you disengage the E-stop and clear the error from the touchscreen.</li>

Reviewer: "from the touchscreen" is overly specific; the instructions above state that you can do it from touchscreen or app.
```suggestion
                    <li>The gripper will automatically place the plate reader's lid back in its docking area after you fully reset the E-stop.</li>
```

                </ul>
            </td>
        </tr>
        <tr>
            <td><strong>Temperature Module</strong></td>
            <td>Heating or cooling is disabled.</td>
        </tr>
        <tr>
            <td><strong>Thermocycler Module</strong></td>
            <td>Heating or cooling is disabled.</td>
        </tr>
        <tr>
            <td><strong>Status light</strong></td>

>             <td>The light turns yellow.</td>

Reviewer: Double-checking this change: it's yellow, not red?

Author: Red. It's red, my mistake. 

I was testing something else, e-stop with no modules and the gantry in a home position. The status bar turns yellow under those conditions.
